src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetLivePairing/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Video } from 'types/tracks';
+import { DashboardVideoLivePairing } from 'components/DashboardVideoLivePairing';
+import { DashboardVideoLiveWidgetTemplate } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetTemplate';
+
+const messages = defineMessages({
+  externalSourcesTitle: {
+    defaultMessage: 'External broadcast sources',
+    description: 'Title used in the external broadcast dashboard widget',
+    id: 'components.DashboardVideoLiveControlPane.widgets.DashboardVideoLiveWidgetLiePairing.title',
+  },
+  exteralSourcesInfo: {
```

Comment:
```suggestion
  info: {
```
This is the id given to the info modal's message displayed in other widgets, and its also what you put in the id :
id: 'components.DashboardVideoLiveControlPane.widgets.DashboardVideoLiveWidgetLiePairing.info'